www/src/components/Sponsors.astro

```diff
@@ -0,0 +1,33 @@
+<style lang="scss">
+.wrapper {
+  margin-bottom: 3rem;
+  margin-top: 3rem;
+}
+
+.list {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+
+  a {
+    &::before,
+    &::after {
+      display: none;
+    }
+  }
+}
+
+.title {
+  font-size: 0.75em;
+  font-weight: 700;
+  text-align: center;
+  margin-bottom: 1.5rem;
+}
+</style>
+
+<div class="wrapper">
+  <h2 class="title">Sponsors</h2>
+  <div class="list">
+    <a href="https://www.netlify.com/?utm_campaign=devex-jl&utm_source=astro&utm_medium=blog"><img src="/assets/netlify.svg" width="147" height="40" alt="Netlify" /></a>
```

Comment:
either remove the utm_campaign from the blog, or update

Reply:
Ah OK I thought the UTM campaign was desired on their part. Will remove.

Reply:
tbh they might like it, but then we should be consistent and accurate. those params are telling them that its coming from our blog, so in that case tell them that its coming from "&utm_medium=homepage" or "&utm_medium=website" instead? tbh I'm still a utm noob.

Reply:
If they didn’t say “use this campaign“ then better to omit. They can always request later.